content/post/053-great-calamity-circumvention-assembly-at-ipfs-camp.md

```diff
@@ -0,0 +1,27 @@
+---
+date: 2019-08-01
+url: 2019-08-01-great-calamity-circumvention-assembly-at-ipfs-camp
+tags: IPFS, IPFS Camp
+title: The Great Calamity Circumvention Assembly at IPFS Camp 2019
+author: Alan Shaw
+---
+
```

Comment:
What about including @alanshaw's PR comment (say, in italics) at the beginning of this post?